config/routes.rb

```diff
@@ -112,5 +112,11 @@
     resources :records, only: [:index, :show]
   end
 
+  namespace :api do
+    namespace :v1 do
+      resources :issues, only: [:index]
+    end
+  end
```

Comment:
Acá me suena mejor algo así:
```suggestion
  scope module: 'api/v1', path: '/api/v1' do
    resources :posts
  end
```
Aunque no se si funciona así, si por algo se complica lo dejamos así, pero si puede quedar en una mejor.

Reply:
este franco si queres ahora que cambie el route chequealo a ver que te parece como lo deje. Seguramente haya alguna mas facil o simple pero tambien esta armada como a futuro

Reply:
Si, ese dejalo así hasta que estemos cerca del merge, no pasa nada. Ahí me fijo si encuentro alguna forma más compacta, sino vamos con el que está y listo.